packages/CollabCodeEdit-Tests.package/CCEProtocolSessionCreationSSTest.class/instance/subClassSpecificSetUp.st

```diff
@@ -0,0 +1,3 @@
+running
+subClassSpecificSetUp
+	
```

Comment:
A comment that says that this has been left empty as it may optionally be specified by a subclass would be helpful